src/tailwind.css

```diff
@@ -84,6 +86,7 @@
   --color-default-app-icon-gradient: var(--ink-default-app-icon-gradient);
 
   /* Typography */
+  --text-*: initial;
```

Comment:
These rules disable the "default" styles included in Tailwind.

We don't really want to use stuff other than what's in the Figma, so I added some of them now